### PR TITLE
Brs

### PR DESCRIPTION
**Pull request by snewz**
Like this?

### CODE REVIEW

File: cangen.c [hunk 1]
```diff
@@ -356,6 +363,8 @@ int main(int argc, char **argv)
 		if (canfd){
 			mtu = CANFD_MTU;
 			maxdlen = CANFD_MAX_DLEN;
+			if(brs)
```
Comment (marckleinebudde) [inline cangen.c]: please add a space after ``if``

File: cangen.c [hunk 2]
```diff
@@ -476,7 +485,9 @@ int main(int argc, char **argv)
 			i = random();
 			extended = i&1;
 			canfd = i&2;
-			rtr_frame = ((i&12) == 12); /* reduce RTR frames to 1/4 */
+			if(canfd)
```
Comment (marckleinebudde) [inline cangen.c]: same here

**Comment by snewz** [comment]
Done.

**Comment by marckleinebudde** [comment]
Oliver, if you give your OK, I'll squash these patches and apply the resulting patch on master. Sven, can I add you Signed-off-by? If so, which e-mail address?

**Comment by marckleinebudde** [comment]
Oh, just noticed that most patches have a S-o-b.

**Comment by hartkopp** [comment]
Still two remarks:

The options -f and -b should be swapped in the help text.
Before case 'b' must be a blank line to follow the current formatting.

When you squashed all that stuff feel free to add my

Acked-by: Oliver Hartkopp <socketcan@hartkopp.net>

Tnx!

**Comment by snewz** [comment]
Options are swapped and blank line is inserted.

**Comment by marckleinebudde** [comment]
squashed and pushed to master. tnx

**Pull request closed without merging**